8.8/clean.sh

```diff
@@ -2,20 +2,20 @@
 
 . variables.sh
 
-echo wiping install area $installdir
+echo "Wiping install area $installdir"
 
 rm -rf $installdir
 mkdir $installdir
 
-echo cleaning checkout area $checkoutdir
+echo "Cleaning checkout area $checkoutdir"
 
 for i in $checkoutdir/*; do
-	if [ -d $i ]; then
-		rm -rf $i
-	fi
+  if [ -d $i ]; then
+    rm -rf $i
+  fi
 done
 
```

Comment:
Now we do out-of-tree builds, we should clean `build` as well as `checkout`. This bit me while flipping between native and docker builds. I added `builddir` to `variables.sh` and added another cleaner here.